app/models/validator_score_v1.rb

```diff
@@ -78,6 +79,12 @@ class ValidatorScoreV1 < ApplicationRecord
     where(data_center_key: data_center_keys)
   end
 
+  def create_commission_history
+    if commission != commission_before_last_save
```

Comment:
This could also be shorter, as condition on callback, like this: `after_save :create_commission_history, if saved_change_to_commission?`. But this will also work.